awscli/customizations/eks/update_kubeconfig.py

```diff
@@ -113,9 +114,9 @@ def _display_entries(self, entries):
 
     def _run_main(self, parsed_args, parsed_globals):
         client = EKSClient(self._session,
-                           parsed_args.name,
-                           parsed_args.role_arn,
-                           parsed_globals)
+                           cluster_name=vars(parsed_args).pop('cluster_name'),
```

Comment:
You wouldn't just use `parsed_args.cluster_name` ?

Reply:
Yes I would. However because I'm passing in `parsed_args` as kwargs later, the init function would receive two (albeit same) values for keyword argument `cluster_name` and results in a TypeError.

Reply:
Fixed in the new changes!